services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -202,6 +205,18 @@ export default class IndexPage extends React.Component<Props> {
   }
 
   renderCommission(commission: Commission) {
-    return <li key={commission.id}>{commission.name}</li>;
+    return (
+      <li style={{ listStyleType: 'none' }}>
+        {' '}
+        <Checkbox
+          id={null}
+          name="commissions"
+          value="commission"
```

Comment:
The `value` needs to be different for each commission so that we know what checkbox is checked.